openedx/features/learner_profile/README.rst

```diff
@@ -0,0 +1,8 @@
+Enterprise Support
```

Comment:
I spent way too long trying to understand the connection between "Enterprise Support" and what I was reading below. :)